discord/user.py

```diff
@@ -698,6 +698,16 @@ async def _get_channel(self):
         ch = await self.create_dm()
         return ch
 
+    @property
+    def mutual_guilds(self):
+        """List[:class:`~discord.Guild`]: Return a list of :class:`~discord.Guild` that the :class:`~User` shares with the bot.
+        .. versionadded:: 1.7
+        .. note::
+
+            This can only return mutual guilds within the client's internal cache.
+        """
+        return [guild for guild in self._state._guilds if guild.get_member(self.id)]
```

Comment:
This is going to AttributeError you need to .values() this

Reply:
Thanks for pointing that out, didnt noticed when i removed the copy